src/OpenApi/src/Transformers/ActivatedOpenApiDocumentTransformer.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.AspNetCore.OpenApi;
+
+internal sealed class ActivatedOpenApiDocumentTransformer(Type transformerType) : IOpenApiDocumentTransformer
```

Comment:
I don't find this type name very intuitive.  Maybe TypeOpenApiDocumentTransformer or something?

Reply:
I wanted to emphasize that this type was used for activated/instantiated transformers. Maybe `TypeInstantiatedOpenApiDocumentTransformer` does the trick? 😅

Reply:
Why is that worth emphasizing?